src/ui/details/detail_view.rs

```diff
@@ -168,9 +198,7 @@ pub fn detail_view(param: DetailView) -> impl View {
 				set_list,
 				env: env.clone(),
 			}),
-			virtual_stack(
-				VirtualDirection::Vertical,
-				VirtualItemSize::Fixed(Box::new(|| 30.0)),
+			dyn_stack(
```

Comment:
Replacing `virtual_stack` with `dyn_stack` is fine as the benefits of the optimizations of `virtual_stack` really only come into play around the thousand item mark and I hope we won't ever have that many fields in an entry